manifests/nginx_vts_exporter.pp

```diff
@@ -72,29 +72,29 @@
 #  [*version*]
 #  The binary release version
 class prometheus::nginx_vts_exporter(
-  String $arch                        = $::prometheus::params::arch,
-  String $bin_dir                     = $::prometheus::params::bin_dir,
-  String $nginx_scrape_uri            = $::prometheus::params::nginx_vts_exporter_nginx_scrape_uri,
-  String $download_extension          = $::prometheus::params::nginx_vts_exporter_download_extension,
-  Variant[Undef,String] $download_url = undef,
-  String $download_url_base           = $::prometheus::params::nginx_vts_exporter_download_url_base,
-  Array $extra_groups                 = $::prometheus::params::nginx_vts_exporter_extra_groups,
-  String $extra_options               = '',
-  String $group                       = $::prometheus::params::nginx_vts_exporter_group,
-  String $init_style                  = $::prometheus::params::init_style,
-  String $install_method              = $::prometheus::params::install_method,
-  Boolean $manage_group               = true,
-  Boolean $manage_service             = true,
-  Boolean $manage_user                = true,
-  String $os                          = $::prometheus::params::os,
-  String $package_ensure              = $::prometheus::params::nginx_vts_exporter_package_ensure,
-  String $package_name                = $::prometheus::params::nginx_vts_exporter_package_name,
-  Boolean $purge_config_dir           = true,
-  Boolean $restart_on_change          = true,
-  Boolean $service_enable             = true,
-  String $service_ensure              = 'running',
-  String $user                        = $::prometheus::params::nginx_vts_exporter_user,
-  String $version                     = $::prometheus::params::nginx_vts_exporter_version,
+  String            $arch               = $::prometheus::params::arch,
+  String            $bin_dir            = $::prometheus::params::bin_dir,
+  String            $nginx_scrape_uri   = $::prometheus::params::nginx_vts_exporter_nginx_scrape_uri,
+  String            $download_extension = $::prometheus::params::nginx_vts_exporter_download_extension,
+  Optional[String] $download_url        = undef,
```

Comment:
This looks good to me, but if you're going to line up the params, I think you need one more space here? Or, alternately, do what most modules do, and don't align the second column (though to me, this actually is easier to read).

I think if it's >= Puppet 4, you can also go back to `$prometheus` vs `$::prometheus` if you like (the former is actually preferred now).

Reply:
thanks for the input. fixed